### DISCUSSION TITLE
Lighhouse scores

### DISCUSSION THREAD

**Discussion by Jean-Baptiste-Lasselle** [swup/swup#771] [General]
Could we improve those results ...?
Would be worth, the library is nice work, thanks

![image](https://github.com/swup/swup/assets/35227860/cb53dadf-9259-4001-9f22-89b7a77e5647)

**Comment by daun** [comment]
Hi @Jean-Baptiste-Lasselle 👋 Absolutely, we agree about keeping performance and best practices in mind. While the demos are not meant to serve as exemplary websites, but rather to showcase specific swup features, there's always room for improvement — do you have any specific suggestions? 😉
